components/v2/delivery/provider/providers/packetery.js

```diff
@@ -39,6 +39,10 @@ const Packetery = ({ errors, delivery, onSelect }) => {
     get(session, DELIVERY_DATA_SESSION_STORAGE_KEY)
   );
 
+  useEffect(() => {
+    setSelectedPickupPoint(get(session, DELIVERY_DATA_SESSION_STORAGE_KEY));
+  },[session])
```

Comment:
@patrikspilka nedame do dependency konkretnejsiu vec z tej session a dorucenie z cart? Potom by sa to neprekreslovalo ak kliknem hocikam, nie?

Reply:
@ivankopcik zapracované ale aj tak sa prekresluje cely komponent keď niečo ine zaškrtneš skušal som všetko zakomentovať a aj tak to robilo tak to sa asi prekresluje lebo sa prekreslí nejaký rodič